### PR TITLE
a new cron

### PR DESCRIPTION
**Pull request by lofty18**

### CODE REVIEW

File: groups/xml-infrastructure/templates/bep_cron.tpl [hunk 1]
```diff
@@ -1,93 +1,108 @@
...
+#LIVE0 11,16 * * * /home/xml/efbackend/produceStats.sh SupportHourlyStatsReport "sbasile@companieshouse.gov.uk,tjohn@companieshouse.gov.uk,mwilliams5@companieshouse.gov.uk,dcornelius@companieshouse.gov.uk,678104@gmail.com" >/dev/null 2>&1
+#LIVE*/1 * * * * /home/xml/errorlogcheck.sh -700 >/dev/null 2>&1
+#LIVE*/10 0-23 * * * /home/xml/efbackend/monitorLCKfilesWarn.sh >/dev/null 2>&1
+#LIVE*/1 * * * * /home/xml/checkformprocess.sh >/dev/null 2>&1
+#LIVE4,34 * * * * /home/xml/efbackend/filinqueuecheck.sh email 100 >/dev/null 2>&1
+#LIVE0 8-17 * * * /home/xmlbeplive/updateXMLFormDetail.sh >/dev/null 2>&1
+#LIVE0 10 * * * /home/xmlbeplive/failedCerts.sh >/dev/null 2>&1
+#LIVE05 10 * * * /home/xmlbeplive/failedMortCerts.sh >/dev/null 2>&1
+#LIVE*/15 7-18 * * * /home/xmlbeplive/efbackend/status7TNEP.sh >/dev/null 2>&1
+#LIVE*/15 7-18 * * * /home/xmlbeplive/efbackend/updateGenBarcodeSub.sh >/dev/null 2>&1
```
Comment (MartyFox) [inline groups/xml-infrastructure/templates/bep_cron.tpl:68]: Just noticed the home directory is incorrect here. Should be `/home/xml`. 
There are a few more in the file as well.

File: groups/xml-infrastructure/templates/bep_cron.tpl [hunk 2]
```diff
@@ -1,93 +1,108 @@
...
+#LIVE#*/1 22-23 * * 5 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE
+#LIVE# Saturday
+#LIVE*/1 0-23 * * 6 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 0-6 * * 6 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE
+#LIVE# Sunday
+#LIVE*/1 0-23 * * 0 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 0-18 * * 0 /home/ewfbeplive/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 20-23 * * 0 /home/ewfbeplive/efbackend/formPartition.sh >/dev/null 2>&1
```
Comment (MartyFox) [inline groups/xml-infrastructure/templates/bep_cron.tpl:30]: Just noticed the home directory is incorrect here. Should be `/home/xml`.

File: groups/xml-infrastructure/templates/bep_cron.tpl (hunk 1 above)
Comment (lofty18) [reply groups/xml-infrastructure/templates/bep_cron.tpl:68]: just updated

File: groups/xml-infrastructure/templates/bep_cron.tpl (hunk 2 above)
Comment (lofty18) [reply groups/xml-infrastructure/templates/bep_cron.tpl:30]: just corrected

**Review by MartyFox — APPROVED** [review]

**Pull request merged**